_includes/parse-server/push-notifications-android.md

```diff
@@ -1,67 +1,48 @@
-#### Configure Broadcast Receiver and Permissions
+#### FCM Push Setup
 
-Add the following service and broadcast receiver definitions to `AndroidManifest.xml` immediately before the *closing* `</application>` tag:
```

Comment:
I think we're also forgetting that we need to add the app to the Firebase console, download the google-services.json and applying the `apply plugin: 'com.google.gms.google-services'` statement too?

Reply:
It is mentioned below, in the "Then, follow Google's docs for setting up an Firebase app" text

Reply:
Yes I found it confusing what to do given there are some steps that aren't necessary.

See: https://github.com/rogerhu/Parse-SDK-Android/blob/3748a0872f9a1263145a9b299210089101b64707/README.md#migrating-to-firebase